example/spec/parallel/kinesisTests/KinesisReplaySpec.js

```diff
@@ -62,7 +62,7 @@ describe('The Kinesis Replay API', () => {
 
   async function cleanUp() {
     // delete rules
-    const rulesToDelete = await rulesList(testConfig.stackName, testConfig.bucket, ruleDir);
+    const rulesToDelete = await readJsonFilesFromDir(ruleDir);
```

Comment:
So you're reading the definitions from the local directory instead of the bucket? Does that introduce a possible problem if not everything from the local directory got synced up to the bucket?

Reply:
They were never being read from the bucket. On master, `rulesList` is calling `setupSeedData` [here](https://github.com/nasa/cumulus/blob/master/packages/integration-tests/index.js#L607). `setupSeedData` was just loading the rules from files [here](https://github.com/nasa/cumulus/blob/master/packages/integration-tests/index.js#L272). The bucket was only being used because `setupSeedData` was passing it to [`setProcessEnvironment`](https://github.com/nasa/cumulus/blob/master/packages/integration-tests/index.js#L251), which was using it to set the `process.env.system_bucket` environment variable, which had no impact on actually returning the list of rules.